src/app/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Navbar.module.css';
+import { MdHome, MdFormatListBulleted, MdPerson } from 'react-icons/md';
+
+export type NavbarProps = {
+  active: 'home' | 'order' | 'user';
+};
+
+const Navbar = ({ active }: NavbarProps): JSX.Element => {
+  const activeLink = {
+    color: 'var(--color-main-bright)',
+  };
+
+  return (
+    <nav className={styles.container}>
+      <Link to="/">
+        <MdHome size={40} {...(active === 'home' ? activeLink : '')} />
```

Comment:
```suggestion
        <MdHome size={40} {...(active === 'home' && activeLink)} />
```
Please try this and check if this works